tests/test_bpsconfig.py

```diff
@@ -66,6 +66,26 @@ def testInvalidArg(self):
             BpsConfig(sequence)
 
 
+class TestBpsConfigGet(unittest.TestCase):
+    def setUp(self):
+        self.config = BpsConfig({"foo": "bar"})
+
+    def tearDown(self):
+        pass
+
+    def testKeyExist(self):
+        """Test if the value is returned when the key is in the dictionary."""
+        self.assertEqual("bar", self.config.get("foo"))
+
+    def testKeyMissing(self):
+        """Test if the provided default is returned if the key is missing."""
+        self.assertEqual(None, self.config.get("baz"))
+
+    def testKeyMissingCustomDefault(self):
+        """Test if the custom default is returned if the key is missing."""
+        self.assertEqual("qux", self.config.get("baz", "qux"))
+
```

Comment:
Just for completeness, I would have added a test when key exists but default given (just to make sure it doesn't always return the default value).  Also, there are no tests where the key isn't global and where the value has env vars or config vars (on one hand these should be tested with search method, but are unit tests written with or without implementation knowledge).  Recommend adding that first default one, then leave it up to you whether to postpone the vars ones to a future ticket.